components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -5036,4 +5036,20 @@ public static boolean isTokenPersistenceEnabled() {
         }
         return OAuth2Constants.DEFAULT_PERSIST_ENABLED;
     }
+
+    /**
+     * Utility method to check if server compatible with client ID tenant unification.
+     *
+     * @return true if compliant.
```

Comment:
without just mentioning compliant, better to have little details when it is compliant and when not

Reply:
Fixed